doc/over_react_redux_documentation.md

```diff
@@ -0,0 +1,289 @@
+# OverReact Redux
+> A Dart React Redux wrapper for targeted state updates.
+
+---
+* __[Purpose](#purpose)__
+    * [Benefits](#benefits)
+* __[Demo Component](#demo-component)__
+* __[Using it in your project](#using-it-in-your-project)__
+* __[ReduxProvider](#reduxprovider)__
+* __[The Connect Function](#the-connect-function)__
+    * [Connect Parameters](#connect-parameters)
+        * [mapStateToProps](#mapstatetoprops)
+        * [mapDispatchToProps](#mapdispatchtoprops)
+        * [mergeProps](#mergeprops)
+        * [areStatesEqual](#arestatesequal)
+        * [areOwnPropsEqual](#areownpropsequal)
+        * [areStatePropsEqual](#arestatepropsequal)
+        * [areMergedPropsEqual](#aremergedpropsequal)
+        * [context](#context)
+        * [pure](#pure)
+        * [forwardRef](#forwardref)
+* __[Using Multiple Stores](#using-multiple-stores)__
+* __[Using Redux DevTools](#using-redux-devtools)__
+
+## Purpose
+[React Redux](https://react-redux.js.org) is a UI binding library that allows for better Redux integration with React. 
+This integration allows for APIs that align more closely with React design patterns while also providing more direct 
+access to store data. OverReact Redux is the Dart wrapper around that library.
+
+### Benefits
+> For a full list of benefits, see <https://react-redux.js.org/introduction/why-use-react-redux>.
+ 
+While there are many benefits of using the library, a primary one is OverReact Redux allows for targeted updates 
+that only update components which receive data that has changed, rather than the whole component tree (as is the 
+behavior with React). By utilizing the `connect()` function in conjunction with `mapStateToProps()`, a component will 
+only update when a piece of information it uses is updated.
+
+## Demo Component
+A basic demo component is available in the `web` directory. In addition to running the app to see how it works, the 
+[store](../web/over_react_redux/store.dart) and [component](../web/over_react_redux/components/counter.dart) have 
+comments that provide additional explanation.
+
+To run and experiment with the demo:
+1. `pub get`
+1. `pub run build_runner serve web`
+1. Navigate to `localhost:8080/over_react_redux`
```

Comment:
```suggestion
1. Navigate to `localhost:8080/over_react_redux/`
```
because your directions don't use webdev_proxy it needs the ending `/` to open :/

Reply:
Good catch!